src/corva/handlers.py

```diff
@@ -20,20 +22,20 @@
 from corva.service.api_sdk import CachingApiSdk, CorvaApiSdk
 from corva.service.cache_sdk import FakeInternalCacheSdk, InternalRedisSdk, UserRedisSdk
 
-StreamEventT = TypeVar('StreamEventT', bound=StreamEvent)
-ScheduledEventT = TypeVar('ScheduledEventT', bound=ScheduledEvent)
+StreamEventT = TypeVar("StreamEventT", bound=StreamEvent)
+ScheduledEventT = TypeVar("ScheduledEventT", bound=ScheduledEvent)
 
 
 def get_cache_key(
     provider: str,
     asset_id: int,
     app_stream_id: int,
     app_key: str,
-    app_connection_id: int,
+    app_connection_ids: List[int],
 ) -> str:
     return (
-        f'{provider}/well/{asset_id}/stream/{app_stream_id}/'
-        f'{app_key}/{app_connection_id}'
+        f"{provider}/well/{asset_id}/stream/{app_stream_id}/"
+        f'{app_key}/{"/".join(map(str, app_connection_ids))}'
```

Comment:
Please don't do this, you need to create 2 caches:
- for target assset
- for rerun asset

and pass those caches inside of the handler.

This includes matching app_connection_id from `app_connection_ids` with one from  `rerun_app_connection_ids` by comparing settings

Reply:
@oleksii-symon-corva-ai  @vladbagmet  Also please consider that we simply can't go with
```py
@scheduled
def lambda_handler:
  pass

@partialmerge
def rerun_handler:
  pass
```
for now as far we have 1 handler == 1 DC app, changing this may be challenging

Reply:
> Please don't do this, you need to create 2 caches:
> 
> * for target assset
> * for rerun asset
> 
> and pass those caches inside of the handler.
> 
> This includes matching app_connection_id from `app_connection_ids` with one from `rerun_app_connection_ids` by comparing settings

I don't know which cache key to use

@ville can you help?

Reply:
the cache key should be eg `corva/well/32842/stream/57326/corva.wits-depth-summary/1434809` where `1434809` is app connection id. 
`"/".join(map(str, app_connection_ids)` simply wont work.
In case of multiple app_connection_ids you need to match source asset's app connections and rerun asset's app connections.